packages/admin-ui/client/index.js

```diff
@@ -1,6 +1,7 @@
 import React, { Suspense, useMemo } from 'react';
 import ReactDOM from 'react-dom';
 import { ApolloProvider } from 'react-apollo';
+import { ApolloProvider as ApolloHooksProvider } from 'react-apollo-hooks'; // FIXME: Use the provided API when hooks ready
```

Comment:
Is this still being used?

Reply:
It is, not sure if that's a problem, though. There's a chain of hooks:

```
useListSearch/useListPagination > useListItems > useListQuery > useQuery
```

I'd like to move away from the render component at some point and this seems to work atm.